lib/hooks/policies/index.js

```diff
@@ -89,6 +86,9 @@ module.exports = function(sails) {
 
 		bindPolicies: function() {
 
+      // Build / normalize policy config
+      this.mapping = this.buildPolicyMap();
```

Comment:
Because here it allow to check policies existence only after all hooks are loaded.
If it will be as now, some hooks wouldn't be able to add policies before this check and application will fail with error: 'no such policy found'